batch7crud-roll10/src/main/java/com/lftechnology/batch7crud/dao/PersonDao.java

```diff
@@ -0,0 +1,37 @@
+package com.lftechnology.batch7crud.dao;
+
+import com.lftechnology.batch7crud.entity.Person;
+import com.lftechnology.batch7crud.utils.DbConnection;
+
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Locale;
+
+/**
+ * @Author binodnme
+ * Created on 1/17/16
+ */
+public class PersonDao {
+    private Connection conn = null;
+    private Statement stmt = null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "stmt" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)